state_transfer/endpoint/route/route.go

```diff
@@ -2,28 +2,36 @@ package route
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/konveyor/crane-lib/state_transfer/endpoint"
 	routev1 "github.com/openshift/api/route/v1"
-	v1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	RouteEndpointTypePassthrough  = "RouteEndpointTypePassthrough"
+	RouteEndpointTypeInsecureEdge = "RouteEndpointTypeInsecureEdge"
```

Comment:
Name change here too?

Reply:
kept this exported to be used in constructor https://github.com/konveyor/crane-lib/pull/45/files#diff-2c8ff7e19854224c2e8023f9580e05010929857caf54373710690e334831032bR66

am I missing something?